xep-0363.xml

```diff
@@ -251,6 +265,7 @@
 <section1 topic='Security Considerations' anchor='security'>
   <ul>
   <li>Client implementors MUST consider the fact that without additional end-to-end-encryption files uploaded to a service described in this document will store those files in plain text on that service. Client implementors SHOULD either use this only for semi public files (for example files shared in a public MUC or a PEP Avatar) or implement appropriate end-to-end encryption.</li>
+  <li>Requesting entities MUST strip any newline characters from the HTTP header names and values before making the PUT request.</li>
```

Comment:
I wonder if we should just say that headers must be canonicalized as described in the HTTP spec? I can't remember if this includes newlines or if it's just the capitalization of the header keys, but I'm pretty sure there's a section in there about that (although I can't find it now).

Reply:
There is already wording in the XEP that the actual upload is out of scope for this document thus repeating (or referencing) HTTP spec is unnecessary IMHO. I see this part of the security section just as a friendly reminder in case some HTTP libraries don't validate the header input.

Reply:
Sounds good to me